console/console-init/ui/src/modules/address-space/dialogs/CreateMessagingProject/MessagingProjectConfiguration.tsx

```diff
@@ -116,23 +86,45 @@ export const Configuration: React.FunctionComponent<IConfiguration> = ({
   ).data || {
     addressSpacePlans: []
   };
-
   if (loading) return <Loading />;
 
   const { namespaces } = data || {
     namespaces: []
   };
+  const onNameSpaceSelect = (value: string) => {
```

Comment:
can you please try keep the consistency for event/functuon name.First event name +postfix.
onSelect+postfix=onSelectNameSpace, onChangStandard etc.

Even we don't need so many function for doing same kind of work. We can create single function and can handle by element name which you can get from event.target.name. Major thing we can avoid to pass down so many functions to child component as props.

const onChangeInput=(value,event)=>{
const elementName=event.target.name;
const newProjectDetail={...projectDetail};
newProjectDetail[elementName]=value;
setProjectDetail(newProjectDetail);
}
wdyt?

Reply:
We can implement this logic in create project